modules/specialized-guides/pages/public-cloud-guide/payg/payg-faq.adoc

```diff
@@ -0,0 +1,254 @@
+
+**Is there a volume discount / tiered pricing built in to the AWS Listing?**:: 
+There is no volume discount available at this time. 
+
+**I no longer require support, how can I end the subscription?**:: 
+If you no longer require support there are 2 options: 
+
+* Unregister all systems from the {productname} Server 
+* Terminate the {productname} instance
+
+== Technical (Billing)  
+
+**Do I need any special or additional infrastructure to setup {productname} billing via AWS Marketplace?**::
+**No**, all components needed to handle the billing are part of the {productname} code-base. 
+
+**What is the deployment mechanism for {productname} on the AWS Marketplace?**:: 
+The AWS Marketplace listing for {productname} is deployed via an (Amazon Machine Image) AMI. 
+
+**What version of {productname} is installed when using the Marketplace listing?**:: 
+The AWS Marketplace listing for {productname} is tied to a specific version of {productname} (4.3.8 at launch).  Future listings will typically include the latest version available at the time AMI creation.  Please check the listing for further information. 
+
+**I need a prior version of {productname} installed, can I still use the listing?**:: 
+**No**. There is no choice over the {productname} version when deploying using the AWS Marketplace listing.  If a prior version of {productname} is required, this will need to be installed manually using the standard documentation. 
+
+**How often is the listing updated (including the version of {productname} etc.)?**:: 
+The Marketplace listing is tied to a specific version of {productname}, usually the latest version available at the time of the listing.
++
+
+Typically, the listing in AWS Marketplace is updated quarterly, or more frequently if there are security issues.  
++
+
+It is worth noting that the image update process may not be fully aligned to the {productname} product maintenance releases (every 6-8 weeks), so {productname} instances should be patched after deployment.  
+
+**I have multiple independent SUSE Manager deployments, each running a separate install of the AWS Marketplace listing for SUSE Manager, how is this billed?**:: 
+As the SUSE Manager deployments are independent, each is billed separately from the next. 
+
+**I have purchased multiple SUSE products from the AWS Marketplace (i.e. Rancher, NeuVector and SUSE Manager), does the AWS Marketplace billing method still work?**:: 
+**Yes**.  The billing mechanism for the various products is independent.  Each deployment will be billed separately via the AWS Marketplace. 
+
+**I already have an existing SUSE Manager deployment, how can I get this deployment to bill via the AWS Marketplace?**:: 
+Existing deployments cannot be directed converted or updated to provide billing via the AWS Marketplace. 
++
+
+It is easy to migrate your existing SUSE Manager deployment to a SUSE Manager instance which has been deployed from and can bill via the AWS Marketplace.
```

Comment:
??? Honestly - I never tried it. I do not know how to migrate all the data from SUMA BYOS to SUMA PAYG.
I think this is what this sentence should say, right?